### PR TITLE
feat(am): add UMA 2.0 functional documentation

### PR DESCRIPTION
**Pull request by tcompiegne**
Closes gravitee-io/issues#4009

### CODE REVIEW

File: pages/am/3.x/protocols/protocols-overview.adoc [hunk 1]
```diff
@@ -18,6 +18,10 @@ OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol.
 It enables Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server,
 as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
 
+== link:/am/current/am_protocols_uma2_overview.html[UMA 2.0]
+
+User-Managed Access (UMA) is an OAuth-based protocol designed to give an individual a unified control point for authorizing who and what can get access to their digital data, content, and services, no matter where all those things live.
```
Comment (lusoalex) [inline pages/am/3.x/protocols/protocols-overview.adoc:23]:
```suggestion
User-Managed Access (UMA) is an OAuth-based protocol designed to give to an individual a unified control point for authorizing who and what can get access to their digital data, content, and services, no matter where all those things live.
```

File: pages/am/3.x/protocols/uma2/overview.adoc [hunk 2]
```diff
@@ -0,0 +1,439 @@
+= UMA 2.0
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_protocols_uma2_overview.html
+:page-folder: am/protocols/uma2
+:page-layout: am
+
+== Overview
+
+User-Managed Access (UMA) is an OAuth-based protocol designed to give an individual a unified control point for authorizing who and what can get access to their digital data, content, and services, no matter where all those things live.
```
Comment (lusoalex) [inline pages/am/3.x/protocols/uma2/overview.adoc:9]:
```suggestion
User-Managed Access (UMA) is an OAuth-based protocol designed to give to an individual a unified control point for authorizing who and what can get access to their digital data, content, and services, no matter where all those things live.
```

File: pages/am/3.x/protocols/uma2/overview.adoc [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+
+== Using UMA 2.0
+
+Gravitee.io AM exposes an endpoint for discovering information about the UMA provider configuration.
+
+NOTE: Discovery endpoint URL : https://am-gateway/{domain}/uma/.well-known/uma2-configuration
+
+You can also find UMA 2.0 protocol endpoints exposed by Gravitee.io Access Management :
+
+* link:/am/current/uma2/index.html[UMA 2.0 API Console, window="_blank"] - link:/am/current/uma2s/swagger.yml[Swagger spec, window="_blank"]
```
Comment (lusoalex) [inline pages/am/3.x/protocols/uma2/overview.adoc:28]:
```suggestion
* link:/am/current/uma2/index.html[UMA 2.0 API Console, window="_blank"] - link:/am/current/uma2/swagger.yml[Swagger spec, window="_blank"]
```

File: pages/am/3.x/protocols/uma2/overview.adoc [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+
+An application that is capable of making requests for protected resources with the resource owner's authorization and on the requesting party's behalf.
+
+* `resource server`
+
+A server that hosts resources on a resource owner's behalf and is capable of accepting and responding to requests for protected resources.
+
+* `authorization server`
+
+A server that protects, on a resource owner's behalf, resources hosted at a resource server.
```
Comment (lusoalex) [inline pages/am/3.x/protocols/uma2/overview.adoc:53]:
```suggestion
resource owner:: An entity capable of granting access to a protected resource, the "user" in User-Managed Access. +
The resource owner MAY be an end-user (natural person) or MAY be a non-human entity treated as a person for limited legal purposes (legal person), such as a corporation.

requesting party:: A natural or legal person that uses a client to seek access to a protected resource. The requesting party may or may not be the same party as the resource owner.

client:: An application that is capable of making requests for protected resources with the resource owner's authorization and on the requesting party's behalf.

resource server:: A server that hosts resources on a resource owner's behalf and is capable of accepting and responding to requests for protected resources.

authorization server:: A server that protects, on a resource owner's behalf, resources hosted at a resource server.
```

Will end up to: 
image

File: pages/am/3.x/protocols/protocols-overview.adoc (hunk 1 above)
Comment (tcompiegne) [reply pages/am/3.x/protocols/protocols-overview.adoc]: FP

File: pages/am/3.x/protocols/uma2/overview.adoc (hunk 3 above)
Comment (tcompiegne) [reply pages/am/3.x/protocols/uma2/overview.adoc]: FP

File: pages/am/3.x/protocols/uma2/overview.adoc (hunk 2 above)
Comment (tcompiegne) [reply pages/am/3.x/protocols/uma2/overview.adoc]: FP

File: pages/am/3.x/protocols/uma2/overview.adoc (hunk 4 above)
Comment (tcompiegne) [reply pages/am/3.x/protocols/uma2/overview.adoc]: FP

**Pull request merged**